src/Fantomas.Tests/PreserveEOLTests.fs

```diff
@@ -516,4 +516,53 @@ let config =
 let config =
     [("n", "1")
      ("d", "2")]
+"""
+
+[<Test>]
+let ``ending with multiline comment should not introduce additional newline`` () =
+    let config: Fantomas.FormatConfig.FormatConfig =
```

Comment:
```suggestion
    let config = { Fantomas.FormatConfig.FormatConfig.Default with PreserveEndOfLine = true }
```

Reply:
Other options than `PreserveEndOfLine` is not important

Reply:
`config` actually already ok on line 7.